1_OCaml/TP/TP1.ipynb

```diff
@@ -20,6 +20,90 @@
     "Puis utilisez ces fonctions sur des arguments de votre choix."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        sqrt((p2-.p1)**2. +. (p2'-.p1')**2.)-. sqrt((p4-.p3)**2. +. (p4'-.p3')**2.) in
</pre><p>Tu peux réutiliser la fonction distance</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>